modules/apps/object/object-api/src/main/java/com/liferay/object/system/SystemObjectDefinitionManager.java

```diff
@@ -47,6 +50,12 @@ public BaseModel<?> deleteBaseModel(BaseModel<?> baseModel)
 	public BaseModel<?> fetchBaseModelByExternalReferenceCode(
 		String externalReferenceCode, long companyId);
 
+	public default Set<String> getAllowedObjectRelationshipTypes() {
+		return SetUtil.fromArray(
```

Comment:
Could you please check if we can create a constant like [this](https://github.com/liferay/liferay-portal/blob/5b3a605b23fef3e99f992190c959c01e33eb72fb/modules/apps/object/object-service/src/main/java/com/liferay/object/internal/action/trigger/util/ObjectActionTriggerUtil.java#L49-L63) in the ObjectRelationshipConstants class and create a method to return the Set like [this](https://github.com/liferay/liferay-portal/blob/5b3a605b23fef3e99f992190c959c01e33eb72fb/modules/apps/object/object-service/src/main/java/com/liferay/object/internal/action/trigger/util/ObjectActionTriggerUtil.java#L29-L31)?

So we can apply this instead create the Set again in other places

Reply:
Perfect. I will created in the `ObjectRelationshipUtil.java`. Thank you @seltong